modules/rosa-sdpolicy-am-billing.adoc

```diff
@@ -4,8 +4,11 @@
 // * rosa_architecture/rosa_policy_service_definition/rosa-service-definition.adoc
 :_mod-docs-content-type: CONCEPT
 [id="rosa-sdpolicy-billing_{context}"]
-= Billing
+= Billing and pricing
 
-{product-title} is billed through Amazon Web Services (AWS) based on the usage of AWS components used by the service, such as load balancers, storage, EC2 instances, other components, and Red{nbsp}Hat subscriptions for the OpenShift service.
+{product-title} is billed directly to your {AWS} account. ROSA pricing is consumption based, with annual commitments or three-year commitments for greater discounting. The total cost of ROSA consists of two components:
 
-Any additional Red{nbsp}Hat software must be purchased separately.
\ No newline at end of file
+* ROSA service fees
+* AWS infrastructure fees
+
+Visit link:https://aws.amazon.com/rosa/pricing/[{product-title} Pricing] page on the AWS website for more details.
```

Comment:
I would either add a "the" in front of the link or remove "page" after it, right now it sounds like it's missing something:
```suggestion
Visit the link:https://aws.amazon.com/rosa/pricing/[{product-title} Pricing] page on the AWS website for more details.
```
```suggestion
Visit link:https://aws.amazon.com/rosa/pricing/[{product-title} Pricing] on the AWS website for more details.
```